Readme.md

```diff
@@ -190,19 +190,60 @@ InterstitialAdManager is a manager that allows you to display interstitial ads w
 #### showAd
 
 Loads an interstitial ad asynchronously and shows it full screen by attaching a view onto the current root view
-controller. 
+controller.
 
 ```js
 InterstitialAdManager.showAd('placementId')
   .then(...)
   .catch(...);
 ```
 
-Promise will be rejected when there's an error loading ads from Facebook Audience network. It will resolve with a 
+Promise will be rejected when there's an error loading ads from Facebook Audience network. It will resolve with a
 `boolean` indicating whether user didClick an ad or not.
 
 **Note:** There can be only one `showAd` call being performed at a time. Otherwise, an error will be thrown.
 
+### BannerView
+BannerView is a component that allows you to display native banners (know as *AdView*). Banners are available in 3 sizes:
+- BANNER_HEIGHT_50
+- BANNER_HEIGHT_90
+- RECTANGLE_HEIGHT_250
+
+You can listen on events:
+- `onAdClick(() => { })`
+- `onAdError(({ nativeEvent: { errorCode: Int, errorMessage: String }}) => { })`
+
+```js
+import { View, StyleSheet } from 'react-native';
+import { BannerView } from 'react-native-fbads';
+
+const onAdClick = () => console.log('Ad clicked!');
```

Comment:
Probably better to name it as `onAdPress` instead of `onAdClick`.

BTW why not just `onPress` and `onError`?